_posts/2020-07-31-kubeflow-1.1-blog-post.md

```diff
@@ -0,0 +1,39 @@
+**Kubeflow 1.1 improves ML Workflow Productivity, Isolation & Security, and GitOps**
+
+The Kubeflow Community's delivery of Kubeflow 1.1 offers users valuable ML workflow automation with Fairing and Kale along with MXNet and XGBoost distributed training operators. It extends isolation and security through the delivery of multi-user pipelines, CVE scanning, and support for Google's Private GKE and Anthos. 1.1 provides a foundation for consistent and repeatable installation and operations using GitOps methodologies powered by blueprints and kpt primitives.
+
+The ML productivity enhancements in 1.1 include end-to-end workflows using Fairing and Kale. The Fairing workflow enables users to build, train and deploy models from a notebook and [Fairing improvements](https://github.com/kubeflow/fairing/releases/tag/v1.0.1) include the support of configuring environment variables and mounting secrets. Fairing also added a config map for a deployer and bug fixes for TensorRTSpec. The workflows enabled by [Kale](https://github.com/kubeflow-kale/kale) include the ability to write model code in a notebook and then automatically build a Kubeflow pipeline that deploys, trains and tunes that model efficiently, using Katib and cached pipeline steps. Kubeflow 1.1 also delivers stable release deliveries of [MXNet and XGBoost operators](https://github.com/kubeflow/common/blob/master/ROADMAP.md), which simplify distributed training on multiple nodes and speeds model creation.
```

Comment:
Perhaps rephrase here since it's weird to see the operators link to the roadmap.